met-web/src/App.scss

```diff
@@ -0,0 +1,6 @@
+@charset "UTF-8";
```

Comment:
Unrelated to this task, should be part of #173

Reply:
Ok will make a separate PR